client/js/app.js

```diff
@@ -70,8 +70,5 @@ function initWithRaven() {
   }
 }
 
-if (document.readyState !== 'loading') {
-  initWithRaven();
```

Comment:
[And you saw me take it out](https://github.com/wellcometrust/wellcomecollection.org/pull/687)